crates/ruma-events/src/call/member/member_state_key.rs

```diff
@@ -16,11 +16,11 @@ pub struct CallMemberStateKey {
 
 impl CallMemberStateKey {
     /// Constructs a new CallMemberStateKey there are three possible formats:
-    /// - "_{UserId}_{DeviceId}" example: "_@test:user.org_DEVICE". `device_id`: Some`, `underscore:
+    /// - "_{UserId}_{DeviceId}" example: "_@test:user.org_DEVICE". `device_id: Some`, `underscore:
```

Comment:
We should probably consistently use `` ` `` instead of `"` because in this first item in the list `_{UserId}_` is rendered in italic without the underscores.